### DISCUSSION TITLE
Disable Notifications from Homebrew-Autoupdate

### DISCUSSION THREAD

**Discussion by amangalampalli** [Homebrew/discussions#2915] [Everyday usage]
`homebrew-autoupdate` is extremely useful. However, is there a way to disable the notifications? I have tried running it with the `--quiet` but that still results in notifications. Is there a possible way to disable notifications from `homebrew-autoupdate`?

**Comment by gromgit** [comment]
I don't use `homebrew-autoupdate` myself, but the documentation seems to indicate that you started it with `brew autoupdate start --enable-notification`. To disable notifications, it seems like you should `brew autoupdate delete` to remove the existing launchd plist, then `brew autoupdate start` to re-enable it without notifications.

**Reply by amangalampalli** [reply]
This didn't seem to work. I had followed your directions exactly, and it seems to result in notifications. I hadn't started it with `enable-notification` flag and it seems that it is enabled by default for some reason. I had deleted the existing config and had restarted the `homebrew-autoupdate` service but it still resulted in notifications. Do you have any other ideas I could try?

**Reply by gromgit** [reply]
@domt4 is probably the best person to advise further, since this is his project.

But because I'm an inveterate nosey-parker, what's the contents of `~/Library/Application\ Support/com.github.domt4.homebrew-autoupdate/brew_autoupdate`?

**Reply by DomT4** [reply]
Simple/quick answer is that post-Catalina there's no need for a third-party dependency to make the notification work, so it is enabled by default. This is noted in the help text and on the GH repo but obviously the flag is a little confusing. The plan is to change the flag at some point to enable an opt-out but as invalid switches fail hard I need to decide on the best way forwards to achieving that without breaking a ton of scripts/etc.

```
      --enable-notification        Send a notification when the autoupdate
                                   process has finished successfully, if
                                   terminal-notifier is installed & found.
                                   Note that currently a new experimental
                                   notifier runs automatically on macOS Catalina
                                   and newer, without requiring any external
                                   dependencies. Must be passed with start.
```

**Reply by gromgit** [reply]
Ah OK, that text is rather vague about what actually happens. I've initiated a PR to clarify the wording: https://github.com/Homebrew/homebrew-autoupdate/pull/80
